react/components/SKUSelector/utils/index.js

```diff
@@ -0,0 +1,91 @@
+import { clone } from 'ramda'
+
+/**
+ * Return the maximum sku price
+ * @param {array of sku's} items 
+ */
+export const getMaxSkuPrice = items => {
+  let maxPrice = 0
+
+  if (items) {
```

Comment:
I think the code is clearer if you invert this logic and return 0 when `!items` and place the reduce call out of the if, to reduce nesting.

Reply:
I disagree with this approach because then we would have more than one return statement

Reply:
And why is that a problem?

Reply:
is this a ⛏ ?

Reply:
@brunojdo sure